packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Branding/index.tsx

```diff
@@ -106,7 +106,7 @@ function Branding({ application }: Props) {
           description="application_details.branding.description"
         >
           <FormField title="application_details.branding.display_name">
-            <TextInput {...register('displayName')} />
+            <TextInput {...register('displayName')} placeholder={application.name} />
```

Comment:
also add placeholder for logo?

Reply:
+1

Reply:
Added. Align with our SIE settings page.